public/js/main.js

```diff
@@ -645,6 +645,37 @@ function alignRightHeaders() {
     });
 }
 
+function addSingleQuoteToCsvIfNeeded(str) {
```

Comment:
a couple of unit tests for this method wouldn’t hurt, just to make sure we’re doing what we think we’re doing.

Reply:
I'm not sure how to add unit test without a lot of additional work. I would need to set up unit tests in the repo. Also, I'm not sure how to access this method, if I export I think I will need extra step to build this file for the browser. Also,  not sure, jest will be happy if I import a method from a file that uses 'document'. etc

Did you have a quick solution in mind that I'm not seeing?

Reply:
I pushed a commit showing how you can:
- make a js file work both in the browser and nodejs (avoid errors related to global "window" and "document")
- export a function globally
- run unit tests in node.js without any framework (nodejs has a unit test framework built in)